src/app/controllers/inputLayout.controller.ts

```diff
@@ -157,9 +157,10 @@ export class InputLayoutController {
     this.keyMap['#'] = '3*';
     this.keyMap['$'] = '4*';
     this.keyMap['N'] = 'NT*';
+    this.keyMap['Delete'] = ' ';
   }
 
-  private inputValue_onKeydown(e: KeyboardEvent) {
+  private async inputValue_onKeydown(e: KeyboardEvent) {
```

Comment:
I don't think you need the `async` here as you are not calling `await` inside the function.